grails-app/views/purchaseOrderWorkflow/purchaseOrder/showOrderItems.gsp

```diff
@@ -219,6 +358,25 @@
           return product && quantity && unitPrice && quantityPerUom && quantityUom
         }
 
+        function validateAdjustmentsForm() {
+
+          var orderAdjustmentType = $("#orderAdjustmentType").val();
+          var description = $("#description").val();
+          var amount = $("#amount").val();
+          var percentage = $("#percentage").val();
+
+          if (!orderAdjustmentType) $("#orderAdjustmentType").notify("Required")
+          if (!description) $("#description").notify("Required")
+          if (!(percentage || amount)) $("#amount").notify("Amount or percentage required")
```

Comment:
Why there are two the same ifs?

Reply:
One is to add error to amount field and the other to percentage field.